### ISSUE TITLE
CSV Export separartor problem

### ISSUE THREAD

**Issue by pepisp** [CymaticLabs/InfluxDBStudio#22]
When exporting data from query to csv all floats have commas which conflicts with separator. So such file is useless as one have to do some extra postprocessing. Example:
01.11.2017 00:00:02,0,02375,0,19375,12 
it should look like:
01.11.2017 00:00:02,0.02375,0.19375,12 
Alternatively you can change the separator to ";", or let user decide what separator to use.

**Comment by meverett** [comment]
I can't reproduce this locally, what language settings are you using? It might be that .NET is choosing to localize numbers with commas instead of decimal points based on localization settings native to the OS.

**Comment by meverett** [comment]
Are you able to build from source? I have a fix in [a new branch](https://github.com/CymaticLabs/InfluxDBStudio/tree/feature/FixNumericExportValues) that you could checkout and build from and see if this fixes the issue. If it does I'll merge into master. I can't reproduce it locally so your help would be appreciated.

When I have more time I'm definitely open to a user selectable delimiter/separator.

**Comment by pepisp** [comment]
You are right that it is due to local float settings as in Poland (I think in whole Europe) we use commas as decimal symbol. Your fix just changed datetime format to : 2018-01-12T08:00:02.0000000Z. But nothing has changed in floats. However when I manually changed decimal symbol in system settings to "." export is then correct.
So a kind of hotfix is to change system setting, but this doesn't fix the problem.

**Comment by meverett** [comment]
I'm working on introducing user editable formats for date, time, decimal/float/double, and integer, as well as CSV delimiter/separator. This will be part of the user settings. It'll take me a little bit to have to time to finish, but I've got it started. I'll let you know when there's an update and maybe you can test again that it resolves your issue. Thanks!

**Comment by pepisp** [comment]
OK, so let me know,when there will be something to test. For now I will use the system settings hack :)
